pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/UserGovernanceDao.sql.stg

```diff
@@ -40,6 +40,12 @@ where participant.guid = :participantGuid
 and s.guid = :studyGuid
 >>
 
+queryGovernancesByParticipantAndStudyIds() ::= <<
+<select_all_governances_with_granted_studies()>
```

Comment:
Woah, cool!  I didn't realize we can do this sort of on-the-fly-concat thing with `.stg` files.